diagnostics.md

```diff
@@ -166,6 +166,17 @@ This test depends on the container engine being healthy (see [check_container_en
 Depending on what part of this check failed, there are various fixes and workarounds. Most however will involve a
 restrictive local network, or an unreliable connection.
 
+
+### check_image_corruption
+#### Summary
+This check tests that container images on the device can be read.
```

Comment:
Really what we are catching here is that each layer that constructs an image has not been corrupted (i.e., matches the checksum we expect).

cc @robertgzr in case there's more to add!